lib/bundler/dsl.rb

```diff
@@ -218,6 +219,14 @@ def method_missing(name, *args)
 
   private
 
+    def gem_helpers
```

Comment:
@segiddins this is to get access to the `generic` method for use as: `gem_helpers.generic(Gem::Platform.local)`. Without turning the platform into a generic platform, you might get something like `"x86_64-linux"` which doesn't exist in the reverse platform map.

I was also contemplating including the `Bundler::GemHelpers` module in the `Bundler::Dsl` class, but that would mean exposing the `generic` method as a public method as part of the DSL, which seemed wrong to me. I opted for the private method to avoid this.

Another thing I was thinking of would be to use another object that already includes `Bundler::GemHelpers` to call `generic` on that, but I don't know which object the DSL has access to that includes the module. Are you aware of such an object so I could nix this method?